### PR TITLE
Add Simply.com support

### PR DESCRIPTION
**Pull request by zraexy**
Add support for the Simply.com [DDNS Service](https://www.simply.com/en/docs/ddns/).

Includes a reversion of #333, and instead uses the alternative fix mentioned in #332.
HTTP 1.1 allows the response to use chunked transfer encoding even if the request did not. Currently chunked transfer is not handled by `fetch_via_socket_io`, which causes response parsing to fail. I wouldn't be surprised if at least one of the existing services was broken by that change.

### CODE REVIEW

File: ddclient.in [hunk 1]
```diff
@@ -2377,7 +2390,7 @@ sub fetch_via_socket_io {
     } else {
         $request .= "https://$server" if defined($proxy);
     }
-    $request .= "/$url HTTP/1.1\n";
+    $request .= "/$url HTTP/1.0\n";
```
Comment (SuperSandro2000) [inline ddclient.in:2393]: I don't think we want to change that for every request.

File: ddclient.in [hunk 2]
```diff
@@ -2387,8 +2400,8 @@ sub fetch_via_socket_io {
     $request .= "User-Agent: ${program}/${version}\n";
     if ($data) {
         $request .= "Content-Type: application/x-www-form-urlencoded\n" if $headers !~ /^Content-Type:/mi;
-        $request .= "Content-Length: " . length($data) . "\n";
     }
+    $request .= "Content-Length: " . length($data) . "\n";
```
Comment (SuperSandro2000) [inline ddclient.in:2404]: Will length be 0 when ``$data`` is empty?

File: ddclient.in [hunk 3]
```diff
@@ -6559,6 +6572,94 @@ sub nic_gandi_update {
...
+        verbose("UPDATE:","updating %s", $h);
+
+        # Set the URL that we're going to update
+        my $url;
+        $url  = "https://$config{$h}{'server'}$config{$h}{'script'}";
+        $url .= "?apikey=$config{$h}{'password'}";
+        $url .= "&domain=$config{$h}{'zone'}";
+        $url .= "&hostname=$h";
+        $url .= "&myip=";
+        $url .= $ip if $ip;
```
Comment (SuperSandro2000) [inline ddclient.in:6624]: What happens if ip is left empty?

File: README.md [hunk 4]
```diff
@@ -33,6 +33,7 @@ Dynamic DNS services currently supported include:
     dinahosting - See https://dinahosting.com
     Gandi       - See https://gandi.net
     dnsexit     - See https://dnsexit.com/ for details
+    Simply.com  - See https://www.simply.com/en/docs/ddns/ for details
```
Comment (toxic0berliner) [inline README.md:36]: this gives me a 404 as of today... Probably merits an update, maybe to this 
```suggestion
    Simply.com  - See https://www.simply.com/en/docs/api/ for details
```

**Comment by SuperSandro2000** [comment]
Closing due to inactivity. Feel free to comment and I can reopen the PR.

**Pull request closed without merging**